go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/UpCloudLtd/upcloud-cli
 go 1.13
 
 require (
-	github.com/UpCloudLtd/upcloud-go-api v0.0.0-20201130170350-7fc02444127a
+	github.com/UpCloudLtd/upcloud-go-api v0.0.0-20210525102709-661419a1956b
```

Comment:
same issue for go sdk pkg version

Reply:
Yeah, this is unfortunately not fixable unless we also change the import paths. We *could* just tag it as `v4.0.0` (instead of `4.0.0`) but that would result in an `incompatible` version tag and I'm not sure that's an actual improvement. see [here](https://golang.org/ref/mod#incompatible-versions)